l10n_nl_bank/readme/CREDITS.rst

```diff
@@ -0,0 +1,4 @@
+Images
+------
+
+* Bank image: <http://www.iconninja.com/banking-icon-466530>
```

Comment:
I can't find license information on this website. Maybe better use the opportunity to get a creative commons one from [the noun project](https://thenounproject.com/search/?q=bank)?

Reply:
Agree

Reply:
thanks!